python/pecos/simulators/quantum_simulator.py

```diff
@@ -61,6 +61,10 @@ def run(self, qops: list[QOp]) -> list:
         """
         meas = []
         for op in qops:
+            if op.metadata is None:
+                op.metadata = {}
+            if self.backend == "state-vector" and op.name == "RZZ" and op.metadata["angles"][0] == 0.0:
+                continue
```

Comment:
I think I would rather have gate transformations (which I think this is essentially) live in the phir -> pypmir conversion and be the same for all simulators, when possible.

Reply:
It might be better to replace it with an "I" gate with the corresponding duration in case an error model or something down the pipeline is using durations to in act idling.